### PR TITLE
Fix warning with logging

### PR DESCRIPTION
**Pull request by Goodmain**
We have the following warning 
```sh
Implicit conversion from float 1721702613.39006 to int loses precision in /app/vendor/ronasit/laravel-helpers/src/Services/HttpRequestService.php on line 187
```

### CODE REVIEW

File: src/HelpersServiceProvider.php [hunk 1]
```diff
@@ -131,14 +116,36 @@ protected function extendRouter()
...
-        Route::macro('versionRange', fn (VersionEnumContract $from, VersionEnumContract $to, $param = null) => $versionRange($from, $to, $param, $this));
-        Route::macro('versionFrom', fn (VersionEnumContract $from, $param = null) => $versionRange($from, null, $param, $this));
-        Route::macro('versionTo', fn (VersionEnumContract $to, $param = null) => $versionRange(null, $to, $param, $this));
-
-        RouteFacade::macro('versionRange', fn (VersionEnumContract $from, VersionEnumContract $to, string $param = null) => $versionRange($from, $to, $param));
-        RouteFacade::macro('versionFrom', fn (VersionEnumContract $from, $param = null) => $versionRange($from, null, $param));
-        RouteFacade::macro('versionTo', fn (VersionEnumContract $to, $param = null) => $versionRange(null, $to, $param));
-
-        RouteFacade::macro('version', fn (VersionEnumContract $version) => RouteFacade::prefix('v' . $version->value));
+        Route::macro('versionRange', fn (
```
Comment (DenTray) [inline src/HelpersServiceProvider.php:119]: let's revert formatting, but try to short lines by making an alias for the VersionEnumContract - `Version`

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude(['docker', 'k8s', 'vendor', 'bootstrap/cache'])
```
Comment (DenTray) [inline .php-cs-fixer.dist.php:4]:
```suggestion
    ->exclude(['vendor', 'bootstrap/cache'])
```

File: .php-cs-fixer.dist.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ->notPath(['_ide_helper.php', '_ide_helper_models.php'])
```
Comment (DenTray) [inline .php-cs-fixer.dist.php:5]:
```suggestion
```

**Review by DenTray — CHANGES_REQUESTED** [review]

File: src/HelpersServiceProvider.php (hunk 1 above)
Comment (Goodmain) [reply src/HelpersServiceProvider.php:119]: Done

File: src/HelpersServiceProvider.php [hunk 4]
```diff
@@ -71,43 +71,28 @@ protected function extendValidator()
...
          * @param string|null $param (default is 'version')
          * @param Route|null $instance
          * @return Router|Route
          */
-        $versionRange = function (?VersionEnumContract $start, ?VersionEnumContract $end, ?string $param, Route $instance = null) {
+        $versionRange = function (
+            ?Version $start,
+            ?Version $end,
+            ?string  $param,
+            Route    $instance = null
```
Comment (DenTray) [inline src/HelpersServiceProvider.php:89]:
```suggestion
            ?Version $start,
            ?Version $end,
            ?string $param,
            Route $instance = null
```

File: src/HelpersServiceProvider.php [hunk 5]
```diff
@@ -131,14 +116,19 @@ protected function extendRouter()
...
+        Route::macro(
+            'versionRange',
+            fn (Version $from, Version $to, $param = null) => $versionRange($from, $to, $param, $this)
+        );
+        Route::macro('versionFrom', fn (Version $from, $param = null) => $versionRange($from, null, $param, $this));
+        Route::macro('versionTo', fn (Version $to, $param = null) => $versionRange(null, $to, $param, $this));
+
+        RouteFacade::macro(
+            'versionRange',
+            fn (Version $from, Version $to, string  $param = null) => $versionRange($from, $to, $param)
```
Comment (DenTray) [inline src/HelpersServiceProvider.php:128]:
```suggestion
            fn (Version $from, Version $to, string $param = null) => $versionRange($from, $to, $param)
```

**Review by DenTray — CHANGES_REQUESTED** [review]

File: src/HelpersServiceProvider.php (hunk 5 above)
Comment (Goodmain) [reply src/HelpersServiceProvider.php]: Done

File: src/HelpersServiceProvider.php (hunk 1 above)
Comment (Goodmain) [reply src/HelpersServiceProvider.php]: Done

File: src/HelpersServiceProvider.php (hunk 4 above)
Comment (Goodmain) [reply src/HelpersServiceProvider.php]: Done

**Review by DenTray — APPROVED** [review]

**Pull request merged**